app/transformations/MemberPaymentsTransformer.scala

```diff
@@ -180,12 +183,31 @@ class MemberPaymentsTransformer @Inject()(
                 )
             }
         }
-        .toList
 
       (memberDetailsWithCorrectVersion, softDeletedMembers) match {
         case (Nil, Nil) => None
         case _ =>
-          val sameMemberPayments: Boolean = userAnswers.sameAs(initialUA, membersPayments, Omitted.membersPayments: _*)
```

Comment:
The initial issue was that we were checking that the member payments to POST is different from the initial ETMP GET data but we didn't take into consideration the 3 fields that also act as control fields (a certain, fixed value infers some behaviour)

To get around this, i've added a smarter json diff function where we can check the resulting diffed values to make sure they do not equal the control value